drivers/display/Kconfig.smartbond

```diff
@@ -0,0 +1,18 @@
+# Smartbond display controller configuration options
+
+# Copyright (c) 2023 Renesas Electronics Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+config DISPLAY_SMARTBOND
```

Comment:
Same comment as below regarding naming of this file/Kconfig symbol- I think including the IP block name here would make sense, to provide future flexibility